Makefile

```diff
@@ -0,0 +1,102 @@
+all: clean build-prod
+
+IMAGE_NAME      ?= "omisego/potterhat:latest"
```

Comment:
Make sure this is consistent with where we’re publishing image to reduce confusions.